### PR TITLE
feat: Use env variable to control the animated spider and the profiler.

### PR DESCRIPTION
**Pull request by cafuego**
So we can turn them off on production.

See https://github.com/UN-OCHA/tools-ansible/pull/989

Refs: OHA-3

### CODE REVIEW

File: config/packages/web_profiler.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 when@dev:
     web_profiler:
-        toolbar: true
+        toolbar: '%env(bool:PLATFORM_ENABLE_PROFILER)%'
```
Comment (attiks) [inline config/packages/web_profiler.yaml:3]: This is only active on dev

File: config/packages/web_profiler.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ when@dev:
 
 when@test:
     web_profiler:
-        toolbar: false
+        toolbar: '%env(bool:PLATFORM_ENABLE_PROFILER)%'
```
Comment (attiks) [inline config/packages/web_profiler.yaml:13]: Same

File: config/packages/api_platform.yaml [hunk 3]
```diff
@@ -2,6 +2,7 @@ api_platform:
     title: 'OCHA API'
     description: 'The public OCHA API'
     version: 0.0.1
+    show_webby: '%env(bool:PLATFORM_SHOW_WEBBY)%'
```
Comment (attiks) [inline config/packages/api_platform.yaml:5]: Better to just disable it

File: .env [hunk 4]
```diff
@@ -14,6 +14,11 @@
 # Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
 # https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
 
+###> ocha/ops ###
+PLATFORM_SHOW_WEBBY=false
```
Comment (attiks) [inline .env:18]: Better to use `.env.prod`

**Review by attiks — CHANGES_REQUESTED** [review]

File: config/packages/api_platform.yaml (hunk 3 above)
Comment (cafuego) [inline config/packages/api_platform.yaml:5]:
```suggestion
    show_webby: false
```

File: .env [hunk 5]
```diff
@@ -14,6 +14,11 @@
...
+PLATFORM_ENABLE_PROFILER=fale
+###< ocha/ops ###
```
Comment (cafuego) [inline .env:20]:
```suggestion
```

File: config/packages/web_profiler.yaml (hunk 1 above)
Comment (cafuego) [inline config/packages/web_profiler.yaml:3]:
```suggestion
        toolbar: true
```

File: config/packages/web_profiler.yaml (hunk 2 above)
Comment (cafuego) [inline config/packages/web_profiler.yaml:13]:
```suggestion
        toolbar: true
```

**Review by attiks — APPROVED** [review]

**Pull request merged**